### PR TITLE
Grand conversion to multi-arch manifests

### PR DESCRIPTION
**Pull request by rn**
This PR converts all packages, test packages and kernels to multi-arch manifests.

All packages have been pushed to hub as -amd64 and -arm64 images and then a manifest is pushed and signed to.

All YAMLs have been updated.

Note, the package build process also changed, and documentation was added.

I've tested with `linuxkit.yml` and it builds on x86_64 and aarch64 but only boots on x86_64 so far.

polar-bear 1

### CODE REVIEW

**Comment by justincormack** [comment]
needs a rebase, sorry

File: docs/packages.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+### Build packages as a maintainer
+
+If you have write access to the `linuxkit` organisation on hub, you
+should also be set up with signing keys for packages and your signing
+key should have a passphrase, which we call `<passphrase>` throughout.
+
+All official LinuxKit packages are multi-arch manifests and most of
+them are available for amd64 and aarm64. Official images *must* be
+build on both architectures and the must be build *in sequence*, i.e.,
```
Comment (justincormack) [inline docs/packages.md]: "and the" -> "and they"

**Comment by rn** [comment]
rebased and fixed typo

File: docs/packages.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+# LinuxKit packages
```
Comment (riyazdf) [inline docs/packages.md]: thanks for adding this :+1:

Comment (rn) [inline docs/packages.md]: I'll add a bunch more documentation about packages. This is just the basics so that folks know how to build

**Comment by rn** [comment]
@arm64b I've tried to run an aarch64 image I build from this with:
```
qemu-system-aarch64 -device virtio-rng-pci -smp 1 -m 1024 -uuid a286556b-cf6f-4b93-b9fe-3756af5783d4 -pidfile linuxkit-state/qemu.pid -kernel linuxkit-kernel -initrd linuxkit-initrd.img -append console=ttyAMA0 -net nic,model=virtio,macaddr=7a:d9:af:93:e2:21 -net user -nographic -machine virt -cpu cortex-a57
```
and there is nothing on the console. Any idea?

**Comment by rn** [comment]
I repushed/signed the `linuxkit/mount` manifest. it was showing errors on `DOCKER_CONTENT_TRUST=1 docker pull`. Why is unclear as it was build just like the other images. The repush was with:
```
DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE="<passphrase>" ./scripts/push-manifest.sh linuxkit/mount:4fe245efb01384e42622c36302e13e386bbaeb08 1
```

I then hit rebuild. let's see where this is going

**Comment by rn** [comment]
amazingly this passed CI. merging before it breaks

**Pull request merged**